types/UIShell/Header.svelte.d.ts

```diff
@@ -29,6 +29,7 @@ export interface HeaderProps
 
   /**
    * Specify the company name
+   * Alternatively, use the named slot "company" (e.g., <span slot="company">...</span>)
```

Comment:
Might want to enclose the HTML example in `` ` ``, as JS doc is commonly interpreted as Markdown.
Would also terminate line above with a `.`, potentially followed by two spaces, which indicates a manual line break.

You e.g. currently get this in VS Code:
![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/834235/30ce6ad9-a22b-4042-9a56-65ce1f9b6c51)